### PR TITLE
Feature/demo bugfix

### PR DESCRIPTION
**Pull request by Asergey91**

### CODE REVIEW

File: app/components/agenda/modal.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export default class AgendaModalComponent extends Component {
       this.toBeDeleted.push(this.currentlyEditing);
       this.isNew = false;
     } else {
-      this.currentlyEditing.rollbackAttributes();
+      // this.currentlyEditing.rollbackAttributes();
```
Comment (lagartoverde) [inline app/components/agenda/modal.js:86]: As I said to arne, I think we should remove the commented code, in order to clean the repo :)

**Review by lagartoverde — APPROVED** [review]
Everything looks good and works 👍

File: app/components/agenda/modal.js (hunk 1 above)
Comment (Asergey91) [reply app/components/agenda/modal.js:86]: I am not sure about this line thats why I left it, rollBackAttributes behaviour is wierd when an empty agendapoint is created. but technically this should be here

File: app/components/agenda/modal.js [hunk 2]
```diff
@@ -104,23 +104,27 @@ export default class AgendaModalComponent extends Component {
...
         const agendapoint = yield this.toBeDeleted[i];
         const behandeling = yield agendapoint.behandeling;
-        const documentContainer = yield behandeling.documentContainer;
-        if(documentContainer){
-          documentContainer.ontwerpBesluitStatus = this.store.findRecord('concept', 'a1974d071e6a47b69b85313ebdcef9f7'); //concept status
-          yield documentContainer.save();
+        if(behandeling){
+          const documentContainer = yield behandeling.documentContainer
+          if(documentContainer){
+            documentContainer.ontwerpBesluitStatus = this.store.findRecord('concept', 'a1974d071e6a47b69b85313ebdcef9f7'); //concept status
```
Comment (abeforgit) [inline app/components/agenda/modal.js:123]: might be good to add a constant somewhere for the status ids

**Review by abeforgit — APPROVED** [review]

**Pull request merged**